### PR TITLE
Updated banner url to an absolute and alt text to be more descriptive

### LINKED ISSUE #124: [BUG] Banner image path is incorrrect
### What is the bug?
Banner image path is incorrect, leading it to be blank on github home.

### How can one reproduce the bug?
Visit Opensearch-project homage on github: https://github.com/opensearch-project

### Screenshot
Screen Shot 2023-01-07 at 11 13 04 AM

### PR DESCRIPTION
**Pull request by dtaivpp**
Signed-off-by: David Tippett <dtip@amazon.com>

### Description
Updated banner url to an absolute and alt text to be more descriptive

Closes #124 

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dtaivpp** [comment]
@dblock got the url fixed here.

**Comment by dtaivpp** [comment]
Also, updated the alt text to be more accessible.

**Review by saratvemulapalli — DISMISSED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![OpenSearch Banner](banner.jpg)
+![OpenSearch logo and name on top of a dark blue background with a slight honeycomb patttern](https://raw.githubusercontent.com/opensearch-project/.github/main/profile/banner.jpg)
```
Comment (dblock) [inline profile/README.md:1]: nice, but did you mean to misspell patttern? :)

**Review by dblock — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (dtaivpp) [reply profile/README.md:1]: You know I read it like three times and missed that. Update inbound in a few

Comment (dtaivpp) [reply profile/README.md]: @dblock Resolved spelling issue.

**Review by dblock — APPROVED** [review]

**Review by peternied — APPROVED** [review]

**Pull request merged**